tensorflow/stream_executor/cuda/cuda_dnn.cc

```diff
@@ -653,6 +653,25 @@ class CudnnFilterDescriptor {
   SE_DISALLOW_COPY_AND_ASSIGN(CudnnFilterDescriptor);
 };
 
+// The errata sheet (JSON format) for marking the cudnn engines that might be
+// buggy. For example, we don't want the engine 999 of forward convolution:
+// R"({ "version" : 1,
+//      "rules"   : [
+//        { "rule_id"             : "ConvFwd_eng999",
+//          "operation"           : "ConvFwd",
+//          "engine"              : 999,
+//          "knob"                : [],
+//          "cudnn_version_start" : 8000,
+//          "cudnn_version_end"   : -1
+//        }
+// ]})"
+// We intentionally return an empty string for now. Alternately, users can also
+// specify an additional errata JSON file via CUDNN_ERRATA_JSON_FILE at runtime.
+std::string CudnnExecutionPlanEngineFilter() {
+  static std::string filter_str = "";
```

Comment:
Recently I added code to not use eng0, to improve startup time. Can you remove my code and incorporate it here?

Reply:
Sure, can you point me with which commit you are referring to? I just want to confirm if we want to avoid the eng0 from all three conv directions: ConvFwd, ConvBwdFilter, ConvBwdData.

Reply:
https://github.com/tensorflow/tensorflow/commit/47f53336e2ade0f31c7b1a74154328cb4c46556a